input/config.json

```diff
@@ -1059,30 +1006,31 @@
       "itemMappings": {
         // Black Lion Exclusives Chest
         "82450": [
+          // Standalone: Other
           16
         ]
       }
     },
     {
       "id": "emote",
-      "name": "Emotes",
+      "name": "Miscellaneous: Emotes",
```

Comment:
I think the category should just be called Emotes. I understand you are trying to separate them from all the other wardrobe based unlocks, but people know what Emotes are - I feel it isn't a distinction that needs to be communicated. 
'Miscellaneous' is also a word I am uncomfortable using in categories (as is Other for that matter) since they run the risk of becoming bucket categories - I know that is less relevant in this case because it is combined with 'Emotes', but then I don't see why it needs to be included at all.

Reply:
I'm trying to separate them from the "hero panel" sections in some way, I'd say emotes (as well as pvp heroes and home instance nodes, if those ever make it) should have some kind of prefix, whatever it is, since they're kinda weird.

Note that all the other sections are part of the hero panel, and are sorted following the hero panel distribution as well, so these feel a bit out of place as is; but yeah, your call :D.

Reply:
I'ld probably base it off their subtitle in the GWU, except none of these things are there either. 🤷 

I think for Emotes nothing additional is needed. If I added heroes, then probably "PvP: Heroes" may help people understand what they are (they seem pretty obscure, given they are for a minor gametype that I'm not sure gets much play). Nodes could be "Home Instance: Nodes" to go with "Home Instance: Cats" if I ever go there. Problem is just that the API for Heroes is rather convoluted, and the API for Nodes is next to useless (only has the ids, and nothing to link them to images, and no way to link them to items so probably need to do that manually). And there's no Cat images at all without reaching into minis.

Reply:
Finishers and mail carriers are classified as "Other Unlocks" in the wardrobe unlocker, so we could just leave it at "Other: Emotes" for the moment.

I wouldn't bother with cats, they're all obtained ingame through special interactions, they're closer to achievements than actual unlocks; plus yeah, no icons, and I don't like the idea of making those up lol.